care/emr/api/viewsets/report/report_upload.py

```
@@ -0,0 +1,215 @@
+
+
+class GenerateReportRequest(BaseModel):
+    template_id: UUID4
+    associating_id: UUID4
+    output_format: str | None = None
+    force: bool = False
+
+    @field_validator("output_format")
+    def validate_output_format(self, value):
+        if value and not GeneratorRegistry.is_registered(value):
+            raise ValueError("Invalid output format")
```

Ruff (0.14.6)

Avoid specifying long messages outside the exception class

(TRY003)